client/src/main/java/org/asynchttpclient/netty/channel/ChannelManager.java

```diff
@@ -488,4 +511,11 @@ public ChannelPool getChannelPool() {
     public EventLoopGroup getEventLoopGroup() {
         return eventLoopGroup;
     }
+
+    public ClientStats getClientStats() {
+        final long totalConnectionCount = openChannels.size();
```

Comment:
Do we need to filter both this count and and the idle channel count below to ensure we don't count instances of `ServerChannel`?

Given that the `ChannelManager` can be set in the config, I imagine someone using Netty as their server *could* pass their `ChannelManager` in here. `openChannels` also explicitly contains two `Set`s, one of which is for server channels.